dist/src/test/java/io/camunda/application/commons/security/CamundaSecurityApplicationYamlTest.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.application.commons.security;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import io.camunda.application.commons.security.CamundaSecurityConfiguration.CamundaSecurityProperties;
+import io.camunda.security.entity.AuthenticationMethod;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Import;
+
+@SpringBootTest(
+    classes = CamundaSecurityApplicationYamlTest.TestConfig.class,
+    properties = {"spring.config.location=file:src/main/config/application.yaml"})
+class CamundaSecurityApplicationYamlTest {
```

Comment:
How about renaming this so that it expresses the purpose of the test better? That could help others understanding it n the future. Something like `DefaultConfigurationTest` (doesn't have to be scoped to security, we can extend it for other configs in the future).

Reply:
We can do that. I'll also move it out of the securitypackage then.